test/elixir/test/proxyauth_test.exs

```diff
@@ -62,11 +62,9 @@ defmodule ProxyAuthTest do
     |> Enum.join("")
   end
 
-  defp hex_hmac_sha1(secret, message) do
+  defp hex_hmac_sha256(secret, message) do
     signature = case :erlang.system_info(:otp_release) do
-      '20' -> :crypto.hmac(:sha, secret, message)
```

Comment:
@nickva I removed the tests for this old erlang versions, because we are on OTP 23+. Is it okay?

Reply:
Good idea!

Reply:
Just make sure to remove the case statement altogether since we don't care about checking the `otp_release`